doc/content/getting-started/migrating/configure-mac-settings.md

```diff
@@ -6,4 +6,6 @@ aliases: "/getting-started/migrating-from-v2/configure-mac-settings"
 
 MAC settings on {{% tts %}} are configurable per end device - see the [MAC settings guide]({{< ref "reference/api/end_device#message:MACSettings" >}}) for instructions.
 
-{{< warning >}} The RX1 delay of end devices is set to 1 second by default. For some end devices, this may lead to downlink messages not being scheduled in time. Therefore, it is recommended that the RX1 delay be increased to 5 seconds. {{</ warning >}}
+{{< warning >}} The RX1 delay of end devices is set to 1 second by default. For some end devices, this may lead to downlink messages not being scheduled in time. Therefore, it is recommended that the RX1 delay be increased to 5 seconds. {{</ warning >}} 
```

Comment:
```suggestion
{{< warning >}} The RX1 delay of end devices is set to 1 sec by default. For some end devices, this may lead to downlink messages not being scheduled in time. Therefore, it is recommended that the RX1 delay be increased to 5 seconds. {{</ warning >}} 
```